### PR TITLE
Enable password read from stdin. Closes 47

### PR DESCRIPTION
**Pull request by vj396**
We need the go client to support password read from stdin, when the field is missing from the config file

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -96,6 +102,9 @@ go_repository(
 
 go_repository(
     name = "com_github_urfave_cli",
+    build_extra_args = ["-exclude=vendor"],
```
Comment (michaeljs1990) [inline WORKSPACE]: Why was this needed, the underlying commit we are pulling shouldn't have changed at all? Same question for the following two lines as well.

File: collins/commands/util.go [hunk 2]
```diff
@@ -69,6 +71,15 @@ func getCollinsClient(c *cli.Context) *collins.Client {
 	}
 
 	collins, err := collins.NewClientFromYaml()
+	if collins.Password == "" {
```
Comment (michaeljs1990) [inline collins/commands/util.go]: What happens when this code is executed and piped from one collins command to the next?

File: WORKSPACE (hunk 1 above)
Comment (vj396) [reply WORKSPACE]: this change was related to something i had on my local environment.

a clean pull removes the need for these lines.

patched

File: collins/commands/util.go (hunk 2 above)
Comment (vj396) [reply collins/commands/util.go]: can you elaborate with an example use case?

Comment (michaeljs1990) [reply collins/commands/util.go]: something like `collins query -n dnsnode | collins set -a test:value`

Comment (vj396) [reply collins/commands/util.go]: it will fail in such a scenario.
i have updated the `README.md` with a note about this fail case
